hack/e2e-suite/services.sh

```diff
@@ -59,7 +59,6 @@ function join() {
 svcs_to_clean=()
```

Comment:
All created services get added to it. See [line 74].(https://github.com/jlowdermilk/kubernetes/blob/e2e/hack/e2e-suite/services.sh#L74)